src/node.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+export default class Node {
+  constructor( element ) {
+    this.element = element,
+    this.link = null
```

Comment:
Nitpick/not required: I prefer using `next` and/or `previous` to name my pointer, because it's a Linked List convention.